src/main/java/it/gov/pagopa/pu/bff/service/HealthIndicatorService.java

```diff
@@ -0,0 +1,62 @@
+package it.gov.pagopa.pu.bff.service;
+
+import it.gov.pagopa.pu.bff.config.MonitoringServiceConf;
+import it.gov.pagopa.pu.bff.model.ServiceStatus;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class HealthIndicatorService {
```

Comment:
this name could be ambigous as the controller
```suggestion
public class CoreHealthIndicatorService {
```